botocore/args.py

```diff
@@ -26,6 +26,13 @@
 import botocore.serialize
 from botocore.config import Config
 from botocore.endpoint import EndpointCreator
+
+# We have moved the PRIORITY_ORDERED_SUPPORTED_PROTOCOLS list out of this file
+# so we import it here to maintain backwards compatibility.  The redundant "as"
+# statement tells tools like Ruff not to remove it
+from botocore.model import (
```

Comment:
Note: importing this from args.py in model.py instead creates a circular import, so we unfortunately need to do it this way.

Reply:
Why does this need to live in `model.py`?

Reply:
It doesn't - moved it to utils.py based on your comment below.  There were more similar constants there, so it made more sense there.